.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+
+# Default ignored files
+/workspace.xml
```

Comment:
I'm not sure what's the thing with the `workspace.xml` and the `.idea. folder.

Reply:
I will take a look and most probably remove this!